src/main/kotlin/app/revanced/patches/youtube/interaction/downloads/resource/patch/DownloadsResourcePatch.kt

```diff
@@ -26,7 +26,7 @@ class DownloadsResourcePatch : ResourcePatch {
     override fun execute(context: ResourceContext): PatchResult {
         SettingsPatch.PreferenceScreen.INTERACTIONS.addPreferences(
             PreferenceScreen(
-                "revanced_downloads",
+                "revanced_downloads_settings",
```

Comment:
By our current convention, we use `revanced_downloads` as the preference screen key and `revanced_downloads_enabled` as the preference key for the switch. For consistency, this should be applied here too.